### PR TITLE
Remove "version" property from "_config.yml" files

### PR DESCRIPTION
**Pull request by lucaswoj**
one step towards #2016 

cc @jfirebaugh @mourner @mollymerp

### CODE REVIEW

**Comment by jfirebaugh** [comment]
https://jekyllrb.com/docs/datafiles/, clever! 👍

File: docs/_layouts/docs.html [hunk 1]
```diff
@@ -27,13 +27,13 @@ <h1 class='space-bottom2'>
 
 <div class='keyline-all fill-darken0 space-bottom round'>
   <div class='pad1y pad2x keyline-bottom strong small clearfix'>
-    <span class='fill-darken1 pad1x round dark inline fr'>mapbox-gl.js {{site.version}}</span>
+    <span class='fill-darken1 pad1x round dark inline fr'>mapbox-gl.js {{site.data.package.version}}</span>
```
Comment (jfirebaugh) [inline docs/_layouts/docs.html]: Probably need to prepend `v` here.

File: docs/_layouts/docs.html [hunk 2]
```diff
@@ -27,13 +27,13 @@ <h1 class='space-bottom2'>
...
     Get started
   </div>
   <div class='pad2'>
     <div class='fill-white js-replace-token' id='head-code'>{% highlight html %}
-<script src='{{site.tileApi}}/mapbox-gl-js/{{site.version}}/mapbox-gl.js'></script>
-<link href='{{site.tileApi}}/mapbox-gl-js/{{site.version}}/mapbox-gl.css' rel='stylesheet' />{% endhighlight html %}
+<script src='{{site.tileApi}}/mapbox-gl-js/{{site.data.package.version}}/mapbox-gl.js'></script>
+<link href='{{site.tileApi}}/mapbox-gl-js/{{site.data.package.version}}/mapbox-gl.css' rel='stylesheet' />{% endhighlight html %}
```
Comment (jfirebaugh) [inline docs/_layouts/docs.html]: And wherever else `{{site.data.package.version}}` is used...

Comment (lucaswoj) [inline docs/_layouts/docs.html]: Good catch. That's something I wouldn't have caught until this hit prod.

Comment (lucaswoj) [inline docs/_layouts/docs.html]: Fixed in the latest commit

**Pull request merged**